services/api/mw2pdf/INSTALL.md

```diff
@@ -1,8 +1,12 @@
 # Installing
 
 This script requires node.js to be installed.
-In order to run the script, you will first need to build the dependencies using `npm install`
+In order to run the script, you will first need to build the dependencies using `yarn install`
 
-## Running the script
+# Building
 
-You can run the script using `node index.js`
\ No newline at end of file
+`yarn build`
+
+# Running
+
+You can run the script using `node built/main.js`
```

Comment:
While we're changing the entry point to the script: lets add a yarn script to run this command (in case it ever changes again), and then update the python server to call `yarn start` or whatever instead of `node index.js`.

This should be a separate commit IMO but should be part of this PR.

Reply:
Great. What do you think about `yarn run` or something like that?

Reply:
`yarn run` sounds perfect!

Reply:
Turns out that `run` is a subcommand of `yarn` that cannot be "hijacked" by naming a script that in `package.json`. The same goes for `exec`. The next best thing I could think of was `main`, so that the user would run `yarn main`. What do you think about this?

Reply:
Turns out that `run` is a subcommand of `yarn` that cannot be "hijacked" by naming a script that in `package.json`. The same goes for `exec`. The next best thing I could think of was `main`, so that the user would run `yarn main`. What do you think about this?

Reply:
`yarn main` feels more python than node somehow / at that point I'd just stick with `yarn start` which also feels awkward but is more standard -- but that's my bikeshed-painting-brain!

`yarn main` totally works!

Reply:
I have changed it so that the command is `yarn start` with aliases to it for `yarn mw2pdf` and `yarn pdf`. Let me know what you think, @slifty .

Reply:
Nice!

Reply:
Resolving given :rocket: